ios/package.json

```diff
@@ -16,7 +16,8 @@
     "Capacitor/Capacitor/",
     "CapacitorCordova/CapacitorCordova/",
     "Capacitor.podspec",
-    "CapacitorCordova.podspec"
+    "CapacitorCordova.podspec",
+    "Capacitor/Capacitor/assets/native-bridge.js"
```

Comment:
same, as it's committed there is no need to add the line since everything on Capacitor/Capacitor/ is already published